### DISCUSSION TITLE
Line drawing differences Ghostty vs. Terminal.app on Linux system

### DISCUSSION THREAD

**Discussion by edrozenberg** [ghostty-org/ghostty#3626] [Issue Triage]
Ghostty Version 1.0.0
Build 8469
Commit 4b4d4062

(MacOS)

I'm seeing line drawing differences when connected (SSH) to a Slackware Linux system and using the updater there which uses I believe `dialog` to present CLI dialogs. I've already added a `/usr/share/terminfo/x/xterm-ghostty` definition to the Linux system.

This is Terminal.app:
![Terminal app on Slackware - Line Drawing](https://github.com/user-attachments/assets/afda64fa-069d-47d7-9533-f88b051cf78e)

This is Ghostty:
![Ghostty on Slackware - Line Drawing](https://github.com/user-attachments/assets/b618b718-78e3-437f-8012-6d1b4c3c6fd1)

**Comment by mitchellh** [comment]
Can you use the terminal inspector to get the codepoints used for the missing lines and verify they're correct?

**Reply by edrozenberg** [reply]
Thanks will give it a try!
